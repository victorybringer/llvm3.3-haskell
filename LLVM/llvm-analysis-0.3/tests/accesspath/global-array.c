int arr[100];

void f() {
  arr[1] = 5;
}
