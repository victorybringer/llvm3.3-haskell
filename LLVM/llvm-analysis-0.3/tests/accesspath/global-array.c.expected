fromList [ ("@f", ("[100 x i32]*", [AccessDeref, AccessArray])) ]